src/components/SideNavBar/SideNavBar.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import '../../styles/Common.scss';
+import LeftPanel from '../../containers/LeftPanel';
+import Header from '../../containers/Header';
+
+class SideNavBar extends Component {
+  render() {
+    const { toggleShowItem, showItem } = this.props;
+
+    const sideBar = (
+      <div className="side-bar">
+        <ul>
+          <li onClick={() => toggleShowItem(1)}>Profile</li>
```

Comment:
I'd use named constants here instead of numbers. In case there's a change or something. You can add a constants file where you store these and other constants, and just import it.

Something like

```
const sideBarItems = {
  sideBar: 0,
  leftPanel: 1, 
  ... and so on
}
```
and then access them by their name, e.g.:  
`sideBarItems.leftPanel`